pkg/core/blockchain.go

```diff
@@ -293,6 +293,11 @@ func NewBlockchain(s storage.Store, cfg config.Blockchain, log *zap.Logger) (*Bl
 		log.Info("TimePerBlock is not set or wrong, using default value",
 			zap.Duration("TimePerBlock", cfg.TimePerBlock))
 	}
+	if cfg.Genesis.TimePerBlock <= 0 {
+		cfg.Genesis.TimePerBlock = defaultTimePerBlock
```

Comment:
We have an old setting as well. Can it be used if `TimePerBlock` is not specified? We obviously need to discontinue the old one, but this way we can make it compatible.

Reply:
Sure.

Reply:
Fixed.